modules/microshift-config-parameters-table.adoc

```diff
@@ -65,27 +65,151 @@ The following table explains {microshift-short} configuration YAML parameters an
+
+|`ingress.tuningOptions.healthCheckInterval: ""`
+|`string` with pattern: `^(0\|([0-9]+(\\.[0-9]+)?(ns\|us\|µs\|μs\|ms\|s\|m\|h))+)$`
+|The default `healthCheckInterval` value is `5s`, which is 5 seconds. This parameter value defines how long the router waits between two consecutive health checks on the router's configured backends. Currently the minimum allowed value is `1s` and the maximum allowed value is `2147483647ms`, which is 24.85 days. Both are subject to change over time.
+
+This value is applied globally as a default for all routes, but can be overridden per-route by the route annotation `router.openshift.io/haproxy.health.check.interval`.
+
+Requires an unsigned duration string of decimal numbers, each with an optional fraction and unit suffix, such as `300ms`, `1.5h` or `2h45m`. Valid time units are `ns`, `us` (or `µs` U+00B5 or `μs` U+03BC), `ms`, `s`, `m`, `h`.
+
+Setting this parameter value to less than `5s` can cause excess traffic due to too frequent TCP health checks and accompanying SYN packet storms.
+
+Setting this parameter value too high can result in increased latency because of backend servers that are no longer available, but have not yet been detected as such.
+
+An empty or `0` value means "no opinion" and the `IngressController` chooses a default which is subject to change over time.
+
+|`ingress.tuningOptions.maxConnections`
+|`integer`, valid values are: `empty`, `0`, `-1`, and the range `2000-2000000`
+|Default value is `0`. defines the maximum number of simultaneous connections that can be established per `HAProxy` process. Increasing this value allows each ingress controller pod to handle more connections at the cost of additional system resources being consumed.
+
+If this field is empty or `0`, the `IngressController` uses the default value of `50000`, but the default is subject to change in future releases.
+
+If the value is `-1` then `HAProxy` dynamically computes a maximum value based on the available resources set with `ulimit` values in the running container. Selecting `-1`, which means `auto`, results in a large value being computed, and therefore each `HAProxy` process incurs significant memory usage compared with the current default of `50000`.
+
+Setting a value that is greater than the current operating system limit prevents the `HAProxy` process from starting.
+//* If you choose a discrete value, for example, `750000`, and the router pod is migrated to a new node, there's no guarantee that that new node has identical ulimits configured. In such a scenario the pod would fail to start. If you have nodes with different ulimits configured (e.g., different\tuned profiles) and you choose a discrete value then the guidance is to use -1 and let the value be computed dynamically at runtime.
+//Does this apply to MicroShift which is single-node?
```

Comment:
not needed ,Microshift is  a single node. so if the number exceeds what OS permits, then  pod will just fail to start.